abstractAgent.py

```diff
@@ -0,0 +1,10 @@
+class abstractAgent:
+
+  def __init__(self):
+    pass
+
+  def getPayoff(self):
```

Comment:
This should be choose right?

Reply:
Also, do we want to follow general naming conventions where only class names are camel case, and methods/functions should be lower case with _ separation? -- not super important